automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/WorkspacePage.scala

```diff
@@ -0,0 +1,13 @@
+package org.broadinstitute.dsde.firecloud.page.workspaces
+
+import org.broadinstitute.dsde.firecloud.page.AuthenticatedPage
+import org.openqa.selenium.WebDriver
+
+
+/*
+A Workspace Page is any page within the workspace (i.e. the Summary tab, Data tab)
+ */
+class WorkspacePage(implicit webDriver: WebDriver) extends AuthenticatedPage {
+  
```

Comment:
Any shared code that could live here? I think all the urls start with `/workspaces/$namespace/$name, right?